DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompletionDate.cshtml

```diff
@@ -68,6 +60,6 @@
       <button class="nhsuk-button" type="submit">Save</button>
     </form>
 
-    <vc:cancel-link asp-controller="@cancelLinkController" asp-action="Index" link-text="Go back" asp-all-route-data="@routeParamsForBackLink" />
+    <vc:cancel-link asp-controller="DelegateProgress" asp-action="Index" link-text="Go back" asp-all-route-data="@routeParamsForBackLink" />

```

Comment:
I think this is incorrect based on what the ticket says, and what was discussed for 836 and the future of the delegate progress page. "Note that these pages should still return the user to the View Delegates page if this page was reached from that page originally."

Reply:
I agree, this does not do what we expect. But, it might not matter too much as we will be sorting these pages out in https://softwiretech.atlassian.net/browse/HEEDLS-862 . So as long as a note is left there to ensure these links are correct, we should be good leaving these incorrect (as long as they build).